vspec/model/constants.py

```diff
@@ -10,127 +10,159 @@
 #
 # Constant Types and Mappings
 #
-
 # noinspection PyPackageRequirements
-from enum import Enum, EnumMeta
-from typing import Sequence, Type, TypeVar
+import re
+import pkg_resources
+from typing import (
+    Sequence, Type, TypeVar, Optional, Dict, Tuple, NamedTuple, Iterator,
+)
+
+import yaml
+
+from vspec.model import CONFIG_PATH, NON_ALPHANUMERIC_WORD
 
 
 T = TypeVar("T")
 
 
-class EnumMetaWithReverseLookup(EnumMeta):
-    """This class extends EnumMeta and adds:
+class VSSConstant(NamedTuple):
```

Comment:
use `class VSSConstant(str):` from https://github.com/GENIVI/vss-tools/pull/81/files, if that one gets in one day (🙏 ), we keep it working

Reply:
Done